src/metadata_admin/urls.py

```diff
@@ -16,7 +16,7 @@
 from drf_spectacular.views import SpectacularSwaggerView
 
 urlpatterns = [
-    path(f'api/metadata/', include('ensembl.production.metadata.admin.api.urls')),
+    path(f'api/ensembl_metadata/', include('ensembl.production.metadata.admin.api.urls')),
```

Comment:
I'd rather keep the `metadata`. In any case this doesn't matter too much since the url are defined by the portal. 
```suggestion
    path(f'api/metadata/', include('ensembl.production.metadata.admin.api.urls')),
```

Reply:
I changed it as it was confusing for me with:
/ensembl_metadata/ 
being the admin pages while the api is:
/api/metadata/
rather than 
/api/ensembl_metadata/
But I don't care that much and will implement it your way. Especially considering that it is defined by the portal.